src/github_utils.py

```diff
@@ -19,48 +19,56 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
-    print(response.text)
+    #print(response.text)
     return response.text
 
 
```

Comment:
The function name `get_valid_lines` is misleading.  The function now returns hunk information and not just line numbers.

```suggestion
Rename to `parse_diff_hunks` for clarity.
```